packages/core/src/lib/assets/createImagesContext.ts

```diff
@@ -1,13 +1,20 @@
-import { v4 as uuid } from 'uuid';
+import { randomBytes } from 'crypto';
+
+import { fileTypeFromBuffer } from 'file-type';
```

Comment:
`file-type` is ESM, which is problematic

`jest` is now passing, but we need the Keystone usage of `esbuild` to work now

Reply:
Vercel/remix e.g. read the package.json of the project to check if `"type": "module"` is set. I guess we can read that and tell the esbuild pipeline to either export the `.keystone/config.js` in CJS or ESM style? 🤔 

https://github.com/keystonejs/keystone/blob/52337e3aa6a57c23c2f0756199a90058a6e5a2ac/packages/core/src/lib/esbuild.ts#L12